docs/source/guidelines/feature_flow.md

```diff
@@ -81,3 +81,11 @@ Ideally we would follow the following QA procedure on each of these environments
 
 #### Checking Performance
 - [ ] Verify that there is no significant performance regression
+
+---
+
+## Tips and tricks for pull request QA testing
```

Comment:
Maybe there are some tips and tricks regarding how to QA efficiently given a feature? For instance when you use `make reset` vs. only rebuilding the service(s) that changed? If I take some of the current PR's as an example, I intuitively would tackle these as follows:

- [Upgrade to Pydantic V2](https://github.com/minvws/nl-kat-coordination/pull/1912):  No feature but affects every service: re-install every service and go through the extended QA flow.
-   [Add more tests for scheduler: external services](https://github.com/minvws/nl-kat-coordination/pull/1891): Mostly added test files in Mula and some renaming, so perhaps rebuild Mula, run every test and verify that tasks can still be created from Rocky
- [add email setting](https://github.com/minvws/nl-kat-coordination/pull/1966): small documentation addition. Rebuild the docs and run them locally, check text for spelling/clarity.
- [Fix loading javascript in Django admin](https://github.com/minvws/nl-kat-coordination/pull/1480): change only affecting the rocky frontend. Rebuild Rocky and verify the UI fix.